st-embroider/2023/2023-12-19.md

```diff
@@ -0,0 +1,79 @@
+# 2023-12-19
+
+Note Taker: Preston
+Time Keeper: Andrey Mikhaylov
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+- [x] Ed Falkner (EF)
+- [x] Chris Manson (CM)
+- [ ] Aaron Chambers (AC)
+- [x] Alex (A)
+- [x] Preston Sego (PS)
+- [ ] Simon Ihmig (SI)
+- [x] Katie Gengler (KG)
+- [ ] Peter Meehan (PM)
+- [x] Andrey Mikhaylov (AM)
+- [ ] Godfrey Chan (GC)
+- [x] Tommy JR (TJ)
+- [ ] heroiceric
+- [ ] Nick Schot (NS)
+- [x] David Taylor 
+- [x] Peter Wagenet
+- [x] Krystan
+- [x] Ignace 
+
+## Topics
+
+### content-tag indi
+
+https://github.com/embroider-build/content-tag/issues/45 
+
+- content-tag _is_ byte ranges
+- will add APIs to content-tag so that users don't need to do slicing
+  - https://github.com/embroider-build/content-tag/issues/39#issuecomment-1832443310
+- potentially add to content-tag:
+  - charStart, charEnd, etc
+  - but wouldn't be needed if there is a `transform` callback capability
+- Krystan volunteers to PR documentation
+
+### GH actions magically turned on, on forks and unknowingly almost published a new unstable when sync-ing the fork 
+
+- when forking, if users have turned actions on by default for their profile/org, they'll need to manually turn actions off for that fork
+- GH problem, GH needs more controls over this
+
+### v2 addons gjs/gts module name exposing system config and bloating bundle size 
+
+- production path includes full system path (sometimes over 300 characters)
```

Comment:
This is a bug we can fix regardless of whether we remove `moduleName`, which I think is also good to do, but the current situation is just a bug: https://github.com/embroider-build/embroider/issues/1592

Reply:
```suggestion
- production path includes full system path (sometimes over 300 characters)
  - issue report: https://github.com/embroider-build/embroider/issues/1592
```